src/OrchardCore/OrchardCore.Users.Abstractions/UserOptions.cs

```diff
@@ -0,0 +1,27 @@
+namespace OrchardCore.Users
+{
+    public class UserOptions
+    {
+        private string _loginPath = "Login";
+        private string _logoffPath = "/Users/LogOff";
+        private string _changePasswordUrl = "ChangePassword";
+
+        public string LoginPath
+        {
+            get => _loginPath;
+            set => _loginPath = value.Trim();
+        }
```

Comment:
Probably here it is best to trim, as we did with the `AdminOptions` 
` _loginPath = value.Trim(' ', '/');